code/game/objects/items/tcg/tcg.dm

```diff
@@ -47,6 +47,34 @@ GLOBAL_LIST_EMPTY(cached_cards)
 	id = temp.id
 	series = temp.series
 
+// This totally isn't overengineered to hell, shut up
+/**
+  * Alright so some brief details here, we store all "immutable" (Think like power) card variables in a global list, indexed by id
+  * This proc gets the card's associated card datum to play with
+  */
+/obj/item/tcgcard/proc/extract_datum()
+	var/list/cached_cards = GLOB.cached_cards[series]
+	if(!cached_cards)
+		return null
+	return cached_cards["ALL"][id]
+
+/obj/item/tcgcard/examine(mob/user)
+	if(flipped)
+		return ..()
+	var/list/examine_data = ..()
+	var/datum/card/data_holder = extract_datum()
+	if(!data_holder)
+		CRASH("A card without a datum has appeared, either the global list is empty, or you fucked up bad. Series{[series]} ID{[id]} Len{[GLOB.cached_cards.len]}")
+	. = list()
+	. += examine_data[1] //WOOOOOOOO THIS CAN ONLY GO WELL
+	. += "Faction: [data_holder.faction]"
+	. += "Cost: [data_holder.summoncost]"
+	. += "Type: [data_holder.cardtype] - [data_holder.cardsubtype]"
+	. += "Power/Resolve: [data_holder.power]/[data_holder.resolve]"
```

Comment:
Add this at the base of the `examine()` to be populated by another proc.

Reply:
@EOBGames want me to clean up my mess?